### PR TITLE
feature: add full moon with face emoji

### PR DESCRIPTION
**Pull request by tcdowney**
[Full Moon With Face](https://emojipedia.org/full-moon-with-face/) is the Mona Lisa of emojis. Why is it smiling? [What does it mean](https://www.pbs.org/treasuresoftheworld/mona_lisa/mlevel_2/mlevel2_smile.html)? 🤔 

🌝👍

Thanks for your consideration.

---

cc @AnthonyRoss

### CODE REVIEW

**Comment by ewrenn8** [comment]
Hello Pair.ist maintainers,

Is there any update on when this PR will be merged, this is a hard blocker for me.

Thanks!
Eli

**Comment by tcdowney** [comment]
@reid47 any updates? This is blocking our release.

**Comment by ewrenn8** [comment]
Hello @reid47,

While reviewing our release, @tcdowney and I have found a sec black vulnerability that can only be solved by this PR. Is there an update on this?

Thanks!
Eli

**Comment by rodolfo2488** [comment]
I need this please

**Review by XanderStrike — APPROVED** [review]
This is a quality open source contribution good job Tim.

File: src/lib/emojis.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export const emojisByName = {
   clock: '🕑',
   check: '✅',
   party_popper: '🎉',
+  full_moon_with_face: '🌝',
```
Comment (tcdowney) [inline src/lib/emojis.js]: Why is there a dangling comma here? This is not Go.

Comment (reid47) [reply src/lib/emojis.js]: That is due to @luan's lint rules!

**Comment by utako** [comment]
Dear @reid47,

I use pair.ist every day, and often find myself lacking the full range of reactions to the lists my colleagues and I make. When I approve or disapprove of something, I can use thumbs up or down. When I like something, I can respond with a heart. I can use the eyes to acknowledge that I've seen something. The rocketship symbolizes both an exploration and a celebration to me. The check mark can mean I've done something. And the party popper is a universally accepted symbol of celebration, though I've never seen a party popper that looks like that. What even is it? Is it a party hat with noise coming out of the part that normally sits on your head? Is it a decorated megaphone? 

What I'm saying is this: there is currently no emoji with which I can express my veiled emotions in a mysterious manner. I want to use :full_moon_with_face: to react to something when I'm not ready to expose my real reaction. I'm allowed the privacy that :full_moon_with_face: enables me to maintain while still interacting with my Pairist community. In fact, I would say that it's a right.

Please merge this PR immediately.

Sincerely,
@utako

**Comment by xoebus** [comment]
This is critical functionality and must be backported as far as possible.

**Comment by reid47** [comment]
@utako Thank you for the thoughtful comment. After careful consideration, my co-maintainers and I realized that without this change, we also have no way to express our veiled emotions in a mysterious manner. We are ready to take the project in this direction.

**Comment by tcdowney** [comment]
We achieved something great yesterday.

First, I'd like to thank everyone for voicing what 🌝 means to them.
Second, I'd like to thank @reid47 for supporting this critical infrastructure.
And lastly, I'd like to thank @AnthonyRoss for starting it all.

225fb5502be21f9b225fb5502be21f9b225fb5502be21f9b225fb5502be21f9b225fb5502be21f9b

**Pull request merged**